src/utils.js

```diff
@@ -59,7 +60,7 @@ async function getConfig(file: string): Promise<ZelConfig> {
  * @param {string} file
  * @return {Promise<T>}
  */
-async function sync(repo: string, branch: string, file: string): Promise<T> {
+async function sync<T>(repo: string, branch: string, file: string): Promise<T> {
```

Comment:
Yup I saw `mri`. May plan on using that since all we really need is a CLI parser.